authorizer_lambda/src/authorizer_lambda/config.py

```diff
@@ -12,6 +18,9 @@ class Config:
     backend_lambda_arn = (
         "arn:aws:lambda:us-east-1:11111111111:function:some-backend-lambda-dev"
     )
+    public_rsa_key = open(
+        os.path.join(root_dir, "id_rsa.pub"), "r"
```

Comment:
If you use pathlib you can just do 

```
keyfile = root_dir / Path('id_rsa.pub')
keyfile.open('r').read()
```

For easier of deployment though, I generally prefer to store the public key as a secret (in the case of AWS, using AWS Secret Manager, but there are other options), and hand them to the lambda through environment variables. In the dev environment you can use `.env` which is also [specified here](https://github.com/araujoyuri/aws_terraform_interview/blob/80635551173bae53362790023d7e0fefe1221502/authorizer_lambda/src/authorizer_lambda/config.py#L14) and [documented with pydantic](https://docs.pydantic.dev/1.10/usage/settings/#dotenv-env-support).

Reply:
I left a comment here saying this field would come from KMS, but I think it was deleted when tinkering with the path lib. Thanks for the tip!